src/main/java/io/jenkins/plugins/coverage/CoverageAction.java

```diff
@@ -55,7 +60,7 @@ private CoverageResult getResult() {
 
         CoverageResult r = null;
         try {
-            r = CoverageProcessor.recoverReport(owner);
+            r = CoverageProcessor.recoverCoverageResult(owner);
         } catch (IOException | ClassNotFoundException e) {
```

Comment:
nit: I'm not a fan of one letter variable names -I'd be tempted to call this `coverageResult` instead of `r` - I find that easier to read for people new to the code base.